pkg/provider/gardener/ruleset/disak8sstig/rules/242400.go

```diff
@@ -228,7 +229,7 @@ func (r *Rule242400) checkKubeProxy(
 	for _, pod := range pods {
 		podTarget := rule.NewTarget("cluster", "shoot", "name", pod.Name, "namespace", pod.Namespace, "kind", "pod")
 
-		rawKubeProxyCommand, err := kubeutils.GetContainerCommand(pod, "kube-proxy")
+		rawKubeProxyCommand, err := kubeutils.GetContainerCommand(pod, kubeProxyContainerNames...)
```

Comment:
```suggestion
		rawKubeProxyCommand, err := kubeutils.GetContainerCommand(pod, kubeProxyContainerNames)
```
After changing kubeProxyContainerNames to a `string`